content/en/docs/user-documentation/virtual-ymca/authentication-providers/daxko-barcode/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Daxko Barcode Authentication
+---
+
+Open Y Gated Content (Virtual Y) [release 0.13](https://github.com/ymcatwincities/openy_gated_content/releases/tag/0.13) includes a new authentication provider to support Daxko Virtual Areas. This will allow associations using Daxko to set up Virtual Area that enable members to access Virtual Y content using only their member barcode!
+
+Instructions for setting up Virtual Areas are in [<i class="fa fa-file-pdf"></i> Daxko's documentation](./Daxko%20Operations%20-%20Virtual%20Check%20In%20-%20Barcode.pdf). If you need assistance configuring Virtual Areas, Daxko's support team can assist you in setup: [support@daxko.com](mailto:support@daxko.com) 
+
+## Configuration
+
+1. Enable `Daxko Barcode Virtual YMCA integration`
+1. OPTIONAL (but highly recommended): configure reCaptcha settings at `/admin/config/people/captcha/recaptcha`.
+1. Add your validation secret and form url and check help messages at `/admin/openy/virtual-ymca/gc-auth-settings/provider/daxkobarcode`.
+1. Save your settings.
+1. Set Daxko Barcode as your main authorization plugin at the Virtual YMCA settings: `/admin/openy/openy-gc-auth/settings`.
+
+Once enabled, the module enables granular configuration to messages that users will receive on the page. It allows changing "Barcode" to something different, like "Member ID", and allows adding help text to assist members in finding their ID. It also allows for global help text to direct members to help channels in case they're unable to log in.
+
+![The Daxko Barcode configuration form](./daxko-barcode-config.png)
+
+Once the module is enabled, members will be presented with the appropriately titled field to log in to Virtual Y.
+
+![The Virtual Y login page with Daxko Barcode authentication and ReCAPCHA enabled](./daxko-barcode-login.png)
```

Comment:
```suggestion
![The Virtual Y login page with Daxko Barcode authentication and reCAPCHA enabled](./daxko-barcode-login.png)
```

Reply:
see https://www.google.com/recaptcha/about/

Reply:
Thanks. I can never remember how to get that right.

Reply:
Nobody can, that's why we have code review in place )))